scripts/release/build-commands/check-environment-variables.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const chalk = require('chalk');
+
+module.exports = () => {
+  if (!process.env.CIRCLE_CI_API_TOKEN) {
+    throw Error(
+      chalk`
+      {red Missing CircleCI API token}
+
+      {white The CircleCI API is used to check the status of the latest commit.}
+      {white This API requires a token which must be exposed via {yellow.bold CIRCLE_CI_API_TOKEN}}
+      {white For instructions on creating this token check out the link below:}
```

Comment:
Suggested wording improvements, based on our IRL convo.:
```
Each person using this script will need to have their own CircleCI API token.
To make this token available to the release script, you can add it to your `bash_profile` like so:

# React release script
export CIRCLE_CI_API_TOKEN=<your-token-here>
```

Reply:
Great! How do you feel about:

![screen shot 2017-10-16 at 2 11 18 pm](https://user-images.githubusercontent.com/29597/31635539-ec981ce2-b27b-11e7-96e4-9b83982691ae.png)